### PR TITLE
Deprecate `Sonata\Doctrine\Bridge\Symfony\Bundle\SonataDoctrineBundle`

### PR DESCRIPTION
**Pull request by phansys**
## Subject
Deprecated `Sonata\Doctrine\Bridge\Symfony\Bundle\SonataDoctrineBundle`.

I am targeting this branch, because this change respects BC.

Triggered by sonata-project/SonataAdminBundle#6173.

## Changelog

```markdown
### Added
- Added `Sonata\Doctrine\Bridge\Symfony\SonataDoctrineBundle`.

### Deprecated
- Deprecated `Sonata\Doctrine\Bridge\Symfony\Bundle\SonataDoctrineBundle` in favor of `Sonata\Doctrine\Bridge\Symfony\SonataDoctrineBundle`.
```

### CODE REVIEW

**Comment by phansys** [comment]
Since the change related to Flex is not working, I removed it in order to just leave the deprecation for  `Sonata\Doctrine\Bridge\Symfony\Bundle\SonataDoctrineBundle`.

**Comment by phansys** [comment]
>is the Sonata\Doctrine\Bridge\Symfony\SonataDoctrineBundle file missing? 

Yes, I just added the missing file. Thank you!

>Maybe also adding an upgrade note?

The upgrade note [was already added](https://github.com/sonata-project/sonata-doctrine-extensions/pull/205/files#diff-64d95dd6a25e4a4746f01d61de729cebR4).

File: src/Bridge/Symfony/SonataDoctrineBundle.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    public function build(ContainerBuilder $container)
+    {
+        $container->addCompilerPass(new AdapterCompilerPass());
+        $container->addCompilerPass(new MapperCompilerPass());
+    }
+
+    public function getPath()
+    {
+        return __DIR__.'/..';
+    }
```
Comment (franmomu) [inline src/Bridge/Symfony/SonataDoctrineBundle.php:33]: Since the config files are now in the same directory, I guess this is not needed or at least it should be changed to just `__DIR__`.

Comment (phansys) [reply src/Bridge/Symfony/SonataDoctrineBundle.php]: You're totally right. Removed.

File: src/Bridge/Symfony/SonataDoctrineBundle.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+{
+    public function build(ContainerBuilder $container)
+    {
+        $container->addCompilerPass(new AdapterCompilerPass());
+        $container->addCompilerPass(new MapperCompilerPass());
+    }
+
+    protected function getContainerExtensionClass()
+    {
+        return SonataDoctrineExtension::class;
```
Comment (jordisala1991) [inline src/Bridge/Symfony/SonataDoctrineBundle.php:32]: Why is this needed, Symfony cant get this class name right? maybe it was because the old structure and now it works, can you try?

Comment (phansys) [reply src/Bridge/Symfony/SonataDoctrineBundle.php]: It's working. Thanks.

**Comment by jordisala1991** [comment]
Thank you @phansys

**Comment by VincentLanglet** [comment]
@phansys Should we do the same with https://github.com/sonata-project/exporter/blob/2.x/src/Bridge/Symfony/Bundle/SonataExporterBundle.php ?

**Comment by phansys** [comment]
>@phansys Should we do the same with https://github.com/sonata-project/exporter/blob/2.x/src/Bridge/Symfony/Bundle/SonataExporterBundle.php ?

Yes, I guess. I'll do the change today or tomorrow.
